# HG changeset patch
# User John W. Eaton <jwe@octave.org>
# Date 1529959957 14400
#      Mon Jun 25 16:52:37 2018 -0400
# Node ID b1aefc73af6e70a8e7fcfc47894d9a4f8184fcc5
# Parent  5f355e5ff344b00b4ce959108bb30201fa70a1b1
use gnulib mkostemps module

* bootstrap.conf (gnulib_modules): Add mkostemps to the list.

diff --git a/bootstrap.conf b/bootstrap.conf
--- a/bootstrap.conf
+++ b/bootstrap.conf
@@ -54,16 +54,17 @@ gnulib_modules="
   largefile
   link
   localcharset
   lstat
   malloc-gnu
   mkdir
   mkfifo
   mkostemp
+  mkostemps
   mktime
   nanosleep
   nproc
   open
   opendir
   pipe-posix
   progname
   putenv
